plugins/ocp/ocp-nvme.c

```diff
@@ -2928,6 +2927,70 @@ static int get_enable_ieee1667_silo(int argc, char **argv, struct command *cmd,
 	return enable_ieee1667_silo_get(dev, cfg.sel, !argconfig_parse_seen(opts, "no-uuid"));
 }
 
+static int enable_ieee1667_silo_set(struct nvme_dev *dev,
+				    struct argconfig_commandline_options *opts)
+{
+	struct ieee1667_get_cq_entry cq_entry;
+	int err;
+	const __u8 fid = 0xc4;
+
+	struct nvme_set_features_args args = {
+		.result = (__u32 *)&cq_entry,
+		.args_size = sizeof(args),
+		.fd = dev_fd(dev),
+		.timeout = NVME_DEFAULT_IOCTL_TIMEOUT,
+		.cdw11 = OCP_SET(argconfig_parse_seen(opts, "enable"), ENABLE_IEEE1667_SILO),
```

Comment:
Please use a variable for the `OCP(...)` part. This is a bit too much in one line.